src/main/spec/internal-billing-api-spec.yaml

```diff
@@ -0,0 +1,75 @@
+openapi: "3.0.2"
+info:
+  title: "rhsm-subscriptions internal billing API"
+  version: 1.0.0
+
+paths:
+  /internal/remittance/accountRemittances:
+    description: 'Operations to get specific account remittances'
+    parameters:
+      - name: accountNumber
+        in: query
+        required: true
```

Comment:
Let's go ahead and make it false, and instead validate that one of `orgId`, `accountNumber` is present in the API controller instead.